docs/usage/types.md

```diff
@@ -189,7 +189,7 @@ _(This script is complete, it should run "as is")_
     pydantic can't validate the values automatically for you because it would require
     consuming the infinite generator.
 
-## Validating the first value
+#### Validating the first value
```

Comment:
This is just a title/indentation error I left in a previous PR and I just noticed it.